index.js

```diff
@@ -517,6 +551,7 @@ function buildArray (schema, code, name, externalSchema, fullSchema) {
           condition += `Array.isArray(obj${accessor})`
           break
         default:
+          // TODO: item.type can be an array of types.
```

Comment:
is this todo related to the commented text?

Reply:
Yes. At this point we need to check if `item.type` is an array and then call into the switch statement from line 531 again for each element in the array. The switch statement should be factored out into a separate function.